### PR TITLE
Bugfix/update plot tars

### PR DESCRIPTION
**Pull request by RoshanShrestha-NOAA**
- WCOSS2 file structure is removed from mesoscale and cam plots tar files, making it clean and easy to navigate plots
- Puerto Rico plots are defined as "prico"
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: scripts/plots/cam/exevs_cam_grid2obs_plots.sh [hunk 1]
```diff
@@ -87,12 +87,13 @@ if [ -d $log_dir ]; then
...
+   cd ${DATA}/${VERIF_CASE}
+   find ./* -name "*.png" -type f -print | tar -cvf ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar -T -
+
 if [ $SENDCOM = YES ]; then
-    find ${DATA}/${VERIF_CASE}/out/*/*/*.png -type f -print | tar -cvf ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar -T -
-    cpreq ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.
-    if [ $SENDDBN = YES ]; then
-        $DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job ${COMOUTplots}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar
-    fi
+    cpreq -v ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.
```
Comment (AliciaBentley-NOAA) [inline scripts/plots/cam/exevs_cam_grid2obs_plots.sh:94]: @RoshanShrestha-NOAA @MarcelCaron-NOAA Will lines like `cpreq -v ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.` still work if you have already cd-ed into ${DATA}/${VERIF_CASE} a few lines above? 

Same question applied in all other scripts that include the new "cd ${DATA}/${VERIF_CASE}" line. Does the path to the .tar file need to change in the cpreq line?

Comment (RoshanShrestha-NOAA) [reply scripts/plots/cam/exevs_cam_grid2obs_plots.sh:94]: Hi @AliciaBentley-NOAA 
The tar files are written to ${DATA}, so, cd ${DATA}/${VERIF_CASE} does not affect the cpreq -v command.

**Comment by AliciaBentley-NOAA** [comment]
Thanks, @RoshanShrestha-NOAA! Just wanted to check that this wasn't a location-relative path and it wouldn't break with the `cd ${DATA}/${VERIF_CASE}` line above! I'm glad that it won't!

**Comment by ShelleyMelchior-NOAA** [comment]
I have cloned your fork and checked out branch `bugfix/update_plot_tars`.
I have set up the cam and mesoscale plots code to run in my environment. I have submitted the 4 cam and 4 mesoscale plots driver scripts.
The logfiles are in `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out`:
```
-rw-r--r-- 1 shelley.melchior emc     1206344 Nov 21 19:16 jevs_mesoscale_headline_plots.o93854228
-rw-r--r-- 1 shelley.melchior emc     1187233 Nov 21 19:17 jevs_mesoscale_snowfall_plots.o93854231
-rw-r--r-- 1 shelley.melchior emc     1188356 Nov 21 19:17 jevs_mesoscale_precip_plots.o93854226
-rw-r--r-- 1 shelley.melchior emc     1171397 Nov 21 19:17 jevs_mesoscale_grid2obs_plots.o93854227
```
```
-rw-r--r-- 1 shelley.melchior emc     1201651 Nov 21 19:23 jevs_cam_headline_plots.o93854773
-rw-r--r-- 1 shelley.melchior emc         180 Nov 21 19:23 jevs_cam_precip_plots.o93854806
-rw-r--r-- 1 shelley.melchior emc     1168028 Nov 21 19:23 jevs_cam_snowfall_plots.o93854761
-rw-r--r-- 1 shelley.melchior emc     1159844 Nov 21 19:23 jevs_cam_grid2obs_plots.o93854766
```
The headline jobs will be done soonest. We can untar those files and see what the resultant unpacking yields.
@RoshanShrestha-NOAA , @MarcelCaron-NOAA

**Comment by ShelleyMelchior-NOAA** [comment]
I untar'd the mesoscale headline tar file. It is better, but still not quite clean. I still see `out/sfc_upper/last90days/` preceding the png files.
```
[clogin02 /lfs/h2/emc/stmp/shelley.melchior/test_tar]$ tar -xvf /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20231120/evs.plots.mesoscale.atmos.headline.v20231120.tar 
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_w.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_e.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_s.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_s.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me_obsge40.dpt_z2.last90days.fhrmean_init00z.buk_conus_w.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.alaska.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_c.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_e.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_w.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.alaska.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_e.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_s.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_c.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_e.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_s.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_c.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_w.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me_obsge30.dpt_z2.last90days.fhrmean_init00z.alaska.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_c.png
./out/sfc_upper/last90days/evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.alaska.png
```

**Comment by ShelleyMelchior-NOAA** [comment]
I think the cd needs to go deeper ... `cd $DATA/${VERIF_CASE}/out/sfc_upper/last90days`, at least in the case for mesoscale headline.

**Comment by RoshanShrestha-NOAA** [comment]
> I think the cd needs to go deeper ... `cd $DATA/${VERIF_CASE}/out/sfc_upper/last90days`, at least in the case for mesoscale headline.

Hi Shelley, we preferred to organize plots in folders out/{group}/{period}/*.png format. This makes a bit easier to navigate through plots when needed. Otherwise, all approximately 20K plots will get untarred at the same folder.

**Comment by RoshanShrestha-NOAA** [comment]
> > I think the cd needs to go deeper ... `cd $DATA/${VERIF_CASE}/out/sfc_upper/last90days`, at least in the case for mesoscale headline.
> 
> Hi Shelley, we preferred to organize plots in folders out/{group}/{period}/*.png format. This makes a bit easier to navigate through plots when needed. Otherwise, all approximately 20K plots will get untarred at the same folder.

Hi @ShelleyMelchior-NOAA, if we don't want any folder, I can do so, let me know.

**Comment by ShelleyMelchior-NOAA** [comment]
Hi Roshan! You can still organize the plots in folders. That is fine. But when you do the tar, please cd into `$DATA/${VERIF_CASE}/out/{group}/{period}/` before the tar. That will allow the untar on emcrzdm to be clean and not retain any predecessor paths.

Are you saying that `out/{group}/{period}/` is what your scripts on emcrzdm are expecting? If that is the case, @AliciaBentley-NOAA , is this a compromise we can make?

**Comment by AliciaBentley-NOAA** [comment]
@RoshanShrestha-NOAA @MarcelCaron-NOAA Please cd at least one directory deeper before making the .tar files. I don't think that /out needs to be in the path when images are untarred. It is implied that we are looking at EVS output.

@malloryprow If the mesoscale and cam paths were reduced to the following on emcrzdm when images were untarred, would that work for copying the precip maps for the precipitation webpage? Thoughts? `./sfc_upper/last90days/evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_s.png`

**Comment by RoshanShrestha-NOAA** [comment]
> Hi Roshan! You can still organize the plots in folders. That is fine. But when you do the tar, please cd into `$DATA/${VERIF_CASE}/out/{group}/{period}/` before the tar. That will allow the untar on emcrzdm to be clean and not retain any predecessor paths.
> 
> Are you saying that `out/{group}/{period}/` is what your scripts on emcrzdm are expecting? If that is the case, @AliciaBentley-NOAA , is this a compromise we can make?

I can adjust emcrzdm scripts accordingly. That is not a limitation.

**Comment by AliciaBentley-NOAA** [comment]
@RoshanShrestha-NOAA @MarcelCaron-NOAA @ShelleyMelchior-NOAA Yes, we request that Roshan and Marcel cd into ${DATA}/${VERIF_CASE}/out/{group}/{period}/ before the making the .tar file so that the tar file only includes image names. I think that all other components of EVSv1 do it this way. 

Bringing in @malloryprow, who initially brought up this issue when trying to make the precipitation map webpage. I believe that Mallory's code on emcrzdm is smart enough to move the untarred images where they need to be on emcrzdm so images are not all in one directory. Others are using this code as a guide to populate their webpages, and I request that Roshan and Marcel use it as a guide as well:
(On emcrzdm: `/home/people/emc/www/htdocs/users/verification/global/gfs/scripts/para_untar_images_wave.py`)

**Comment by malloryprow** [comment]
The global_det wave and global_det atmos grid2grid each reside in their own image directory. There were too many images from the global_det atmos grid2obs to keep them all together in one directory on RZDM (though this would have been my preferred way to do it to keep it all organized).

**Comment by ShelleyMelchior-NOAA** [comment]
I have brought in the updates and deleted the headline restart dirs for cam and mesoscale and am re-running the headline plots.
The logfiles in `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out`:
```
-rw-r--r-- 1 shelley.melchior emc     1221064 Nov 21 20:53 jevs_mesoscale_headline_plots.o93860801
-rw-r--r-- 1 shelley.melchior emc     1212075 Nov 21 20:54 jevs_cam_headline_plots.o93860800
```

**Comment by ShelleyMelchior-NOAA** [comment]
Yesss! mesoscale headline works as requested.
```
[clogin02 /lfs/h2/emc/stmp/shelley.melchior/tar_out_meso]$ tar -xvf /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20231120/evs.plots.mesoscale.atmos.headline.v20231120.tar 
evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_w.png
evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_e.png
evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_s.png
evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_s.png
evs.mesoscale.bcrmse_me_obsge40.dpt_z2.last90days.fhrmean_init00z.buk_conus_w.png
evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.alaska.png
evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_c.png
evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_e.png
evs.mesoscale.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_w.png
evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.alaska.png
evs.mesoscale.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_e.png
evs.mesoscale.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_s.png
evs.mesoscale.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_c.png
evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_e.png
evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_s.png
evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_c.png
evs.mesoscale.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_w.png
evs.mesoscale.bcrmse_me_obsge30.dpt_z2.last90days.fhrmean_init00z.alaska.png
evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_c.png
evs.mesoscale.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.alaska.png
```

**Comment by ShelleyMelchior-NOAA** [comment]
And the same for cam headline, as expected.
```
[clogin02 /lfs/h2/emc/stmp/shelley.melchior/tar_out_cam]$ tar -xvf /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/cam/atmos.20231120/evs.plots.cam.atmos.headline.v20231120.tar 
evs.cam.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.hawaii.png
evs.cam.bcrmse_me_obsge60.dpt_z2.last90days.fhrmean_init00z.hawaii.png
evs.cam.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init06z.alaska.png
evs.cam.bcrmse_me.tmp_z2.last90days.fhrmean_init06z.alaska.png
evs.cam.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_e.png
evs.cam.bcrmse_me_obsge40.dpt_z2.last90days.fhrmean_init00z.buk_conus_w.png
evs.cam.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_w.png
evs.cam.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_s.png
evs.cam.bcrmse_me.dpt_z2.last90days.fhrmean_init06z.alaska.png
evs.cam.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_c.png
evs.cam.bcrmse_me_obsge30.dpt_z2.last90days.fhrmean_init06z.alaska.png
evs.cam.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_c.png
evs.cam.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_s.png
evs.cam.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_e.png
evs.cam.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_s.png
evs.cam.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.hawaii.png
evs.cam.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_w.png
evs.cam.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.buk_conus_c.png
evs.cam.bcrmse_me.dpt_z2.last90days.fhrmean_init00z.buk_conus_w.png
evs.cam.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_s.png
evs.cam.bcrmse_me.tmp_z2.last90days.fhrmean_init00z.hawaii.png
evs.cam.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_c.png
evs.cam.bcrmse_me.ugrd_vgrd_z10.last90days.fhrmean_init00z.buk_conus_e.png
evs.cam.bcrmse_me_obsge50.dpt_z2.last90days.fhrmean_init00z.buk_conus_e.png
```
Tests for mesoscale precip plotting will happen before the PR is approved for merge.

**Comment by ShelleyMelchior-NOAA** [comment]
@RoshanShrestha-NOAA , I pulled in your changes and am re-running mesoscale precip plots. I see the change in JEVS_CAM_PLOTS, too. Should I re-run a cam plots process, too? I don't expect any behavior change by removing the errant `fi`. I could re-run cam headline plots which is quick.

`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_precip_plots.o93915474`

**Comment by RoshanShrestha-NOAA** [comment]
> @RoshanShrestha-NOAA , I pulled in your changes and am re-running mesoscale precip plots. I see the change in JEVS_CAM_PLOTS, too. Should I re-run a cam plots process, too? I don't expect any behavior change by removing the errant `fi`. I could re-run cam headline plots which is quick.
> 
> `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_precip_plots.o93915474`

The 'fi' in CAM j-job was throwing out an error in my test run. Since there was no if statement above, we don't need that 'fi' there. Yes, you may re-run cam headline and precip plots. Some inconsistencies in naming of CAM plots were discovered and Marcel suggested me to include them in the ongoing PR. 

You may use vhr=14 for meso plots and vhr=22 for cam plots so that all precip maps are generated properly. I see the meso plots jobs completed, but the precip maps are not generated. Probably, the vhr was defaulted to 00. The headline plots job are good.

**Comment by ShelleyMelchior-NOAA** [comment]
Ok, got it. I just kicked off meso precip plots for vhr=14. I just kicked off cam precip plots for vhr=22.
```
                                                                                                   Req'd  Req'd   Elap
Job ID                         Username        Queue           Jobname         SessID   NDS  TSK   Memory Time  S Time
------------------------------ --------------- --------------- --------------- -------- ---- ----- ------ ----- - -----
93924680.cbqs01                shelley.melchi* dev             jevs_mesoscale*   188849   12  1536    --  04:00 R 00:01
93924907.cbqs01                shelley.melchi* dev             jevs_cam_preci*   214513    4   512    --  05:30 R 00:00
```
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_precip_plots.o93924680`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_precip_plots.o93924907`

**Comment by RoshanShrestha-NOAA** [comment]
> Ok, got it. I just kicked off meso precip plots for vhr=14. I just kicked off cam precip plots for vhr=22.
> 
> ```
>                                                                                                    Req'd  Req'd   Elap
> Job ID                         Username        Queue           Jobname         SessID   NDS  TSK   Memory Time  S Time
> ------------------------------ --------------- --------------- --------------- -------- ---- ----- ------ ----- - -----
> 93924680.cbqs01                shelley.melchi* dev             jevs_mesoscale*   188849   12  1536    --  04:00 R 00:01
> 93924907.cbqs01                shelley.melchi* dev             jevs_cam_preci*   214513    4   512    --  05:30 R 00:00
> ```
> 
> `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_precip_plots.o93924680` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_precip_plots.o93924907`

@ShelleyMelchior-NOAA, the mesoscale precip plot job is complete, however, I did not see precip maps being generated. I suspect the needed hourly precip stats files are not ready for it yet when the job was kicked off. Yesterday night, in my test run all precip maps were generated as expected. So, I think, if we re-run the job in a while, it should plot the maps. Other than this, everything looks good.
Screen Shot 2023-11-22 at 9 36 38 AM

**Comment by ShelleyMelchior-NOAA** [comment]
Ok, thanks @RoshanShrestha-NOAA . How about I re-run for vhr=14 but use an older VDATE? My run used 20231121. I will use 20231120.

**Comment by RoshanShrestha-NOAA** [comment]
> Ok, got it. I just kicked off meso precip plots for vhr=14. I just kicked off cam precip plots for vhr=22.
> 
> ```
>                                                                                                    Req'd  Req'd   Elap
> Job ID                         Username        Queue           Jobname         SessID   NDS  TSK   Memory Time  S Time
> ------------------------------ --------------- --------------- --------------- -------- ---- ----- ------ ----- - -----
> 93924680.cbqs01                shelley.melchi* dev             jevs_mesoscale*   188849   12  1536    --  04:00 R 00:01
> 93924907.cbqs01                shelley.melchi* dev             jevs_cam_preci*   214513    4   512    --  05:30 R 00:00
> ```
> 
> `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_precip_plots.o93924680` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_precip_plots.o93924907`

@ShelleyMelchior-NOAA , the mesoscale precip plot job is complete, but it was unable to produce precip maps. As it ran for I suspect the needed hourly precip stats jobs were not complete when this job was kicked off. 

> Ok, thanks @RoshanShrestha-NOAA . How about I re-run for vhr=14 but use an older VDATE? My run used 20231121. I will use 20231120.

Yes, that should work.

**Comment by MarcelCaron-NOAA** [comment]
Removing the directory structure from the files in the plots tarballs looks good to me!  Thanks

**Comment by MarcelCaron-NOAA** [comment]
Also @ShelleyMelchior-NOAA,  
I asked @RoshanShrestha-NOAA if he can add a minor edit of two plots scripts to this PR.  [Here is that commit](https://github.com/NOAA-EMC/EVS/pull/361/commits/072a457cfc74260446db0244d8423da262ea94fd).  These are changes to correct a minor issue in the filenames of asnow plots.

To test these changes, the following job will be sufficient:
jevs_cam_snowfall_plots

**Comment by ShelleyMelchior-NOAA** [comment]
Ok, I have the test runs underway for mesoscale precip and cam snowfall plotting.
meso precip is for 20231120 at vhr=14.
cam snowfall is for 20231120 at vhr=00.
I deleted the associated restart directories that were generated during yesterday's runs for 20231120.
```
93929780.cbqs01                shelley.melchi* dev             jevs_mesoscale*   139324   12  1536    --  04:00 R 00:20
93931068.cbqs01                shelley.melchi* dev             jevs_cam_snowf*   250849    4   512    --  05:30 R 00:01
```
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_precip_plots.o93929780`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_cam_snowfall_plots.o93931068`

**Comment by RoshanShrestha-NOAA** [comment]
@ShelleyMelchior-NOAA , the mesoscale precip job is completed and it looks all good. We now have precip maps as expected.

**Comment by MarcelCaron-NOAA** [comment]
✔️ @ShelleyMelchior-NOAA the cam snowfall plots job completed, no issues in the stdout file, and the output tarball looks good—specifically (1) asnow file names look good to me and (2) files are still structured properly in the tarball.

**Comment by ShelleyMelchior-NOAA** [comment]
Excellent. I will do the formal code review and approve this PR for merge.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Changes look good. Tests yield expected results. This PR is approved for merge.

**Pull request merged**